class/cortex-tenant-ns-label.yml

```
@@ -0,0 +1,11 @@
+parameters:
+  kapitan:
+    compile:
+      - input_paths:
+          - cortex-tenant-ns-label/component/app.jsonnet
+        input_type: jsonnet
+        output_path: apps/
+      - input_paths:
+          - cortex-tenant-ns-label/component/main.jsonnet
+        input_type: jsonnet
+        output_path: cortex-tenant-ns-label/
```

This should use `${_base_directory}` instead of the hard-coded component name for new components:

```suggestion
      - input_paths:
          - ${_base_directory}/component/app.jsonnet
        input_type: jsonnet
        output_path: apps/
      - input_paths:
          - ${_base_directory}/component/main.jsonnet
        input_type: jsonnet
        output_path: cortex-tenant-ns-label/
```